src/test/scala/nl.knaw.dans.easy.validatebag/EasyValidateDansBagServletSpec.scala

```diff
@@ -57,9 +57,26 @@ class EasyValidateDansBagServletSpec extends TestSupportFixture
       resultMessage.profileVersion shouldBe 0
       resultMessage.infoPackageType shouldBe SIP
       resultMessage.isCompliant shouldBe false
-      resultMessage.ruleViolations.value.toList should contain inOrderOnly(
+      resultMessage.ruleViolations.value.toList should contain (
```

Comment:
Since you removed the `inOrderOnly`, what else does it contain now that we're not aware of? Please **don't cut corners**, keep the `inOrderOnly` and list all errors that come from this validation. Also, if you want to prove a rule is not listed here, please do something like `resultMessage.ruleViolations.value.mkString("\n") should not include "1.3.1(a)"`

Reply:
(y)